### DISCUSSION TITLE
Error while running build script after initialising the project.

### DISCUSSION THREAD

**Discussion by megabyte0x** [Autonomous-Finance/ao-starter-kit#9] [Q&A] [answered]
I just created a project with **basic lua template** and tried to follow the steps defined in the documentation. 

## Steps to Recreate the issue

1. `npx create-ao-dapp@latest`
2. Selected `Basic lua` template.
3. Selected `PermawebDeployment` as a feature
4. Install dependencies using `npm i`
5. Give the permissions to run build script.
```bash
chmod +x ./ao/backend/scripts/build.sh
```
6. Tried to run build `bun run backend:build`

Got the following errors:
```bash
./scripts/build.sh: line 7: luacheck: command not found
./scripts/build.sh: line 13: amalg.lua: command not found
```

## What I have tried

1. Tried to install **luacheck** using `luarocks install luacheck` but its giving me an error: `zsh: command not found: luacheck
`

Thanks in advance for the help!

**Comment by parthks** [comment, marked as answer]
need to install amalg -> luarocks install amalg
don't know about luacheck. Been following https://github.com/Autonomous-Finance/teal-ao-starter and everything works. 

luarocks install luacheck should work according to https://github.com/mpeterv/luacheck
